pkg/beholder/auth.go

```diff
@@ -2,26 +2,75 @@ package beholder
 
 import (
 	"crypto/ed25519"
+	"encoding/binary"
 	"fmt"
+	"time"
 )
 
 // authHeaderKey is the name of the header that the node authenticator will use to send the auth token
 var authHeaderKey = "X-Beholder-Node-Auth-Token"
 
 // authHeaderVersion is the version of the auth header format
 var authHeaderVersion = "1"
+var authHeaderVersionV2 = "2"
```

Comment:
nit/ VersionVersion? :shrug: 
```suggestion
var authHeaderVersion2 = "2"
```